edsm.py

```diff
@@ -30,70 +30,132 @@ def __init__(self):
     def link(self, system_name):
         self.cancel_lookup()
         if system_name in self.FAKE:
-            self.result = { 'img': '', 'url': None, 'done': True, 'uncharted': False }
+            self.result = { 
```

Comment:
While we can fairly easily see what you've done this is noise that detracts from the real code changes, it's code style that doesn't really fit with any tools either I think.